pkg/crc/machine/machine.go

```diff
@@ -63,17 +64,18 @@ func getCrcBundleInfo(bundlePath string) (*bundle.CrcBundleInfo, error) {
 	return bundle.Extract(bundlePath)
 }
 
-func loadCrcBundleInfo(startConfig *StartConfig, bundleName string) (*bundle.CrcBundleInfo, error) {
-	crcBundleMetadata, err := bundle.GetCachedBundleInfo(bundleName)
-	if err != nil {
-		return nil, err
+func getBundleMetadataFromDriver(driver drivers.Driver) (string, *bundle.CrcBundleInfo, error) {
+	bundleName, err := driver.GetBundleName()
```

Comment:
@cfergeau So now we are doing it because all the driver config now have bundle info right?